File: Inf-121/POO/def2_inf121/d2_e3_Veterinaria/E3_Veterinaria/Principal.cs
using System;
namespace E3_Veterinaria
{
	public class Principal
	{
		public static void Main(){
			Console.WriteLine("\n\n\tDatos Estudiante: Cristhian Andres Escobar Herrera");
			Console.WriteLine("\tCI: 6908109 L.P.");
			//a) 
			Veterinaria v1 = new Veterinaria();
			Veterinaria v2 = new Veterinaria("vet cuida");
			Propietario p1 = new Propietario();
			// b) 
			v1.mostrar();
			v2.mostrar();
			p1.mostrar();
			p1.mostrar("13579");
			v1.mostrar("Miraflores");
			//c) 
			v1 = v1 > p1; 
			v1.mostrar();
			//d)
			Console.ReadKey();
		}	
	}
}

File: Inf-121/POO/def1_inf121/E2_Supermercado/src/Principal.java
public class Principal {

	public static void main(String[] args) {
		System.out.println("");
		System.out.println("\tDatos Estudiante: Cristhian Andres Escobar Herrera");
		System.out.println("\tCI: 6908109 L.P.");
		// A) Instanciar 2 supermercados de forma distinta

		Supermercado s1 = new Supermercado();
		Supermercado s2 = new Supermercado("Easy Mart");

		// b) Mostrar los 2 supermecados

		s1.mostrar();
		s2.mostrar();

		// c) mostrar el supermercado con mas stock disponible del producto de nombre x
		s1.mayStock(s2, "leche");
		
		// d) Mostrar el nombre de los productos que tiene el supermercado de nombre Y,
		// que son más baratos en el supermercado de nombre X.


		// e) Implementar un método para verificar si el supermercado tiene stock
		// disponible del producto de nombre X.
		s1.vStock("pasta");
		s2.vStock("jabon");

	}

}
